conda_recipes/README.md

```diff
@@ -26,7 +26,7 @@ Now, we will go ahead and build the conda package of TC and all of its dependenc
 
 ```Shell
 cd /
-git clone http://www.github.com/facebookresearch/TensorComprehensions --recursive
```

Comment:
any particular reason why we should clone your fork now?

Reply:
leftover from  building the conda packages, thanks!